README.md

```diff
@@ -1,76 +1,66 @@
-<BR>
+## What is OpenEBS?
 
-## Important documents and contacts
+OpenEBS is an open-source Container Native Storage solution that provides persistent storage for Kubernetes workloads. It enables dynamic provisioning of storage resources using containerized storage controllers, making it highly flexible and cloud-native. OpenEBS supports various storage engines, including LocalPVs for direct node storage and Replicated PV advanced data replication and resilience. It is desiganed to integrate seamlessly with Kubernetes, offering benefits like storage policies, reize, thin-provisiong, snapshots, and restore capabilities, making it an ideal choice for stateful applications. To read more about OpenEBS, check [here](https://github.com/openebs/openebs/).
 
-This repository contains all the project files for the organization, and the following project files are important for understanding the OpenEBS project, and how we operate:
+## Who we are?
 
-* [OpenEBS Vision](/VISION.md) Defines the vision, mission statement; what is in-scope and out-of-scope for the project
-* [Governance](/GOVERNANCE.md) How the project operates
-* [Contributing](/CONTRIBUTING.md) How to contribute, guidelines for being a contributor
-<BR>
+We are a community of innovative Data Storage engineers that contribute to this project. OpenEBS is the result of hundreds of amazing individuals, contributors and storage engineers who provide brainstorming ideas, feedback, code reviews and high-quality code to the project. Anyone who is passionate about storage and data is welcome here. <BR>
 
-## Project Leadership team
+## Community
 
-This Community is managed by the OpenEBS ```Maintainers``` and <kbd>**Senior leaders**</kbd> within the OpenEBS project team. We liaise with the Linux Foundation and CNCF on project, governance
-and operational matters. We curate the daily operations of the project, product, roadmaps, initiatives, all engineering/code activities and all events (including conferences).
-<BR>
+This Community repository is the ```central location``` for project info and files like our governance, contribution, code of conduct, security and vision for the OpenEBS organization laid out by the Maintainers. You'll also find a [Top level Issues List](https://github.com/openebs/openebs/issues) for Organization related issues that span across products/projects. We also provide a [main Discussions forum](https://github.com/openebs/openebs/discussions) where **```everyone```** can collaborate and discuss anything.
 
-## Who we are
+- Homepage: [openebs.io](https://openebs.io/)
+- Mailing list: openebs-team@googlegroups.com
+- Slack: [openebs slack](https://kubernetes.slack.com/messages/openebs)
```

Comment:
nit: just because below on twitter we have it as simply openebs 

```suggestion
- Slack: [openebs](https://kubernetes.slack.com/messages/openebs)
```